services/core/MongodbHistorian/tests/test_prod_query_mongo.py

```diff
@@ -0,0 +1,367 @@
+import pytest
+from  datetime import datetime
+#building name replaced. replace before testing
```

Comment:
So the topic would be BUILDNG1 not Building1.  But since we are case insensitive this works as written?